app/ports/templatetags/sort_version.py

```diff
@@ -5,4 +5,4 @@
 
 @register.filter('sortversion')
 def sortversion(distribution, key):
-    return sorted(distribution, key=lambda x: ((0, ) if x[key] == "none" else tuple(int(i) for i in x[key].split("."))))
+    return sorted(distribution, key=lambda x: ((0, 1) if x[key] == "" else (0, 2) if x[key] == "none" else tuple(int(i) for i in x[key].split("."))))
```

Comment:
Is it not possible to use the `LooseVersion` logic here as well, similar to port utilities?

Reply:
Oh, right. Thanks. Got missed by me totally.